feconf.py

```diff
@@ -570,6 +570,7 @@ def get_empty_ratings():
 SUGGESTION_LIST_URL_PREFIX = '/suggestionlisthandler'
 SUGGESTION_URL_PREFIX = '/suggestionhandler'
 SUBSCRIBE_URL_PREFIX = '/subscribehandler'
+TRAINED_CLASSIFIER_MODEL_HANDLER = '/ml/trainedclassifierhandler'
```

Comment:
I still don't think we're using this (I think I mentioned this in a previous review). Let's either use it, or remove it.